lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -282,11 +283,16 @@ def max_thread_limit
   repositories.each {|remote, opts| opts[:thread].join }
 
   fixtures("symlinks").each do |source, target|
-    if is_windows
-      fail "Cannot symlink on Windows unless using at least Puppet 3.5" if !puppet_symlink_available
-      Puppet::FileSystem::exist?(target) || Puppet::FileSystem::symlink(source, target)
+    unless File.symlink(target)
```

Comment:
Did you mean `File.symlink?` here?

This may have been fixed in later Ruby versions, but IIRC, symlink support was pretty broken in Ruby on Windows in earlier versions.